backend/src/test/java/com/benine/backend/http/RequestHandlerTest.java

```diff
@@ -42,6 +42,29 @@ public final void testRespond() throws Exception {
     verify(out).close();
   }
 
+  @Test
```

Comment:
You should hover make sure at the end of every test method that out is emptied. This could otherwise result in quirky behaviour.